rdkPlugins/LocalTime/source/LocalTimePlugin.cpp

```diff
@@ -83,6 +85,54 @@ bool LocalTimePlugin::postInstallation()
     return true;
 }
 
+// -----------------------------------------------------------------------------
+/**
+ *  @brief postInstallation OCI hook.
+ *
+ *  If set_tz parameter is set then its value should be a path to file.
```

Comment:
It is interesting that in schema file you are naming it `setTZ` and here `set_tz` and it works. But it seams that files from schema doesn't like capital letters :). What matters is that it works.